v22.1/restore.md

```diff
@@ -106,6 +106,11 @@ You can exclude a table's row data from a backup using the [`exclude_data_from_b
 - All [tables](create-table.html) (which automatically includes their [indexes](indexes.html))
 - All [views](views.html)
 
+{% include_cached new-in.html version="v22.1" %} Also, a full cluster restore will:
+
+- Restore [temporary tables](temporary-tables.html) to their original database during a full cluster restore.
```

Comment:
Is it worth mentioning that previously they would always be restored to `defaultdb`?